### DISCUSSION TITLE
Watch specific labels, pull requests edition

### DISCUSSION THREAD

**Discussion by cor3ntin** [community/community#66309] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Github allows to subscribe to specific labels, which is a fantastic feature, incredibly useful for large repos.
However that is only an option for issues, and not PRs

<img src="https://github.com/community/community/assets/1409019/b35ad626-5579-49ce-98db-cc1c96557dd1" width="250">

It would love that feature to be extended to pull requests.
I think it make sense that people would want to subscribe to PR or to issues independently, so keeping the options separate would be the most flexible.

**Comment by Savio-Sou** [comment]
How did you enable the label filter?

Doesn't seem to be showing up on my end:

<img width="310" alt="image" src="https://github.com/community/community/assets/72797635/d32d69a5-5e30-4a6e-a89e-c89495f74ad7">

**Comment by cor3ntin** [comment]
@Savio-Sou you first need to check the "issues" checkbox and it should show up

**Comment by Savio-Sou** [comment]
Thanks @cor3ntin, doesn't seem like that's the case though 🤔 (perhaps GitHub has put us on A/B tests)

<img width="306" alt="image" src="https://github.com/community/community/assets/72797635/7491127f-e199-4be2-b33c-30a5f9975713">
